examples/discovery-search-app/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This file was created by https://github.com/bahmutov/add-typescript-to-cypress, but it seems like some of these settings are covered in `cypress/tsconfig.json`. I'm guessing we could kill one of those files, and just hang on to the superset of configs.

Reply:
yeah i think it makes sense to move this into the cypress directory. im worried leaving it in `discovery-search-app` will cause confusion that the search app is written in typescript, which it is not